contrib/gorm.io/example_test.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Please read https://github.com/DataDog/dd-trace-go/tree/v1/contrib#usage to better understand how to name files and folder structure.